dataland-frontend/src/components/forms/parts/elements/basic/CheckboxesListFormElement.vue

```diff
@@ -107,6 +105,5 @@ export default defineComponent({
       default: "",
     },
```

Comment:
Testing is missing for this "bug".

I will add some test code now to cover it for the future.

Reply:
(Can you pls review the test code on Monday ?)

Reply:
Thanks for test. Their are detailed

Reply:
Ok will resolve this now then